jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
This is an interesting topic, so parentId in thrift model should be set only if reference is `childOf` https://github.com/uber/jaeger/blob/master/model/converter/json/from_domain.go#L82

What if there are multiple `childOf` references? Which one to choose? First one/second...? Inject operation is affected too?  

What if there is only one `followsFrom`/`whatEverFrom`. Should it be this set in thrift as parent. Should it be injected?